src/main/java/Jake.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Scanner;
+
+public class Jake {
+
+    // List out all tasks
+    private static void listTask(Task[] commands, int totalCommands) {
+        System.out.println("Current commands being executed: ");
+        for (int i=0; i<totalCommands; i++){
```

Comment:
I'd like to suggest to you to add at least a space after variable names, for example `for (int i = 0; i < totalCommands; i++`. This makes the code more readable.